foreman/data_refinery_foreman/foreman/management/commands/update_downloadable_samples.py

```diff
@@ -0,0 +1,9 @@
+import django.contrib.postgres.fields
+from data_refinery_common.models import Experiment, ComputationalResultAnnotation
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        for experiment in Experiment.objects.all():
+            organism_ids = list(ComputationalResultAnnotation.objects.filter(data__is_qn=True).values_list('data__organism_id', flat=True))
```

Comment:
can we put `organism_ids` outside of the `for-loop`?

Reply:
Oh snap good call!